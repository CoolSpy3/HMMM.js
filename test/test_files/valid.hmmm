0  read  r1
1  calln r14 7
2  write r2    # Here's a comment about  this particular line
3  halt
4  nop
# This, on the other hand, is a full-line comment
5  nop
6  nop

7  jltzn r1 14
8  jeqzn r1 14
9  setn  r2 1
10 jeqzn r1 15
11 mul   r2 r2 r1
12 addn  r1 -1
13 jumpn 10
14 setn  r2 -1
15 jumpr r14
