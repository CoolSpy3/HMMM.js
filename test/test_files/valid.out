0000 0001 0000 0001
1011 1110 0000 0111
0000 0010 0000 0010
0000 0000 0000 0000
0110 0000 0000 0000
0110 0000 0000 0000
0110 0000 0000 0000
1111 0001 0000 1110
1100 0001 0000 1110
0001 0010 0000 0001
1100 0001 0000 1111
1000 0010 0010 0001
0101 0001 1111 1111
1011 0000 0000 1010
0001 0010 1111 1111
0000 1110 0000 0011
